src/components/toolbar.scss

```diff
@@ -1,16 +1,32 @@
+@import '../common/variables.scss';
 
 .dw-toolbar {
   @apply bg-navy-blue p-0 !important;
   .v-toolbar__content{
     @apply p-0 !important;
   }
+ 
   .theme--light.v-btn:not(.v-btn--icon):not(.v-btn--flat),
   .theme--dark.v-btn:not(.v-btn--icon):not(.v-btn--flat){
     @apply border-white text-white border-solid;
     @apply bg-transparent;
     height: 34px !important;
     border-radius: 4px;
   }
+  .theme--light.v-btn.v-btn--disabled {
```

Comment:
could you repeat this disabled pattern in the dw-toolbar__light section with dude-blue text?